Form-Controls/styles.css

```diff
@@ -0,0 +1,83 @@
+/* General */
+
+html {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+*,
+*:before,
```

Comment:
```suggestion

```

Do you need to set box sizing rules for pseudo content when you haven't defined any pseudo content?